symfony/phpunit-bridge/5.3/phpunit.xml.dist

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- https://phpunit.readthedocs.io/en/latest/configuration.html -->
+<!-- If you use phpunit-bridge directly, set -->
+<!-- xsi:noNamespaceSchemaLocation="bin/.phpunit/phpunit.xsd" -->
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
+         backupGlobals="false"
+         colors="true"
+         bootstrap="tests/bootstrap.php"
+>
+    <php>
+        <ini name="error_reporting" value="-1" />
+        <server name="APP_ENV" value="test" force="true" />
+        <server name="SHELL_VERBOSITY" value="-1" />
+        <!-- If you use phpunit-bridge directly, you can control extra behavior -->
+        <!--
+        <server name="SYMFONY_PHPUNIT_REMOVE" value="" />
+        <server name="SYMFONY_PHPUNIT_VERSION" value="8.5" />
```

Comment:
what about defining `9.5` as value?
related to https://phpunit.de/supported-versions.html

Reply:
`8.5` is the safer default because it works for all PHP versions that Symfony 5 can be run on.